source/extensions/filters/http/lua/config.cc

```diff
@@ -16,9 +16,9 @@ Http::FilterFactoryCb LuaFilterConfig::createFilterFactoryFromProtoTyped(
     const std::string& stat_prefix, Server::Configuration::FactoryContext& context) {
   auto& server_context = context.serverFactoryContext();
 
-  FilterConfigConstSharedPtr filter_config(
-      new FilterConfig{proto_config, server_context.threadLocal(), server_context.clusterManager(),
-                       server_context.api(), context.scope(), stat_prefix});
+  FilterConfigConstSharedPtr filter_config(new FilterConfig{
+      proto_config, server_context.threadLocal(), server_context.clusterManager(),
+      server_context.api(), context.scope(), stat_prefix, server_context.bootstrap()});
```

Comment:
You can get the node metadata from `server_context.localInfo().node().metadata()` directly. You needn't to copy the bootstrap file.